### ISSUE TITLE
Compare two columns data in custom filter

### ISSUE THREAD

**Issue by FranciscoHanna** [aslagle/reactive-table#480]
Hi! First of all, great package!

I was trying to make a custom filter that would compare two columns. Is there anyway that I could achieve this?

I'm working with the `checkboxFilter` example. When I check the box I want to display only those where the value of column 1 is greater than the value of column 2.

**Comment by aslagle** [comment]
Sorry, the filtering's only set up to work with one column at a time.

**Comment by FranciscoHanna** [comment]
Yes, I went trough the code and realized that. I achieved what I want it by denormalizing this comparison upon insert.

Thanks anyway and great package!

**Comment by aslagle** [comment]
Glad you figured something out!
